sign.cddl

```diff
@@ -1,6 +1,6 @@
 COSE-Sign-coswid<payload> = [
     protected: bstr .cbor protected-signed-coswid-header1,
-    unprotected: unprotected-signed-coswid-header,
+    unprotected: unprotected-sign-coswid-header,
```

Comment:
Please check this.
It looked like the unprotected-sign-coswid-header construction was otherwise unused, but I'm also not sure how much interaction across CDDL snippets we intend.

Reply:
it should be "signed" consistently. So we'll abandon this PR, but I'll push a type-names branch after this.
(this is a late reply, but we were stuck in another "branch" of the publication process)